service/src/test/java/bio/terra/policy/service/region/RegionServiceTest.java

```diff
@@ -74,33 +74,33 @@ void locationContainsRegionNegative() {
 
   @Test
   void getRegionsForLocationInvalidRegion() {
-    var result = regionService.getRegionsForLocation("invalid", GCP_PLATFORM);
+    var result = regionService.getRegionCodesForLocation("invalid", GCP_PLATFORM);
```

Comment:
Why add "code"? What is the difference between a region and a region code?

Reply:
The Region object has a code which is different than its id. I am trying to be clear what is being returned. Returning the Region itself is probably clearer. All these Strings are confusing anyway.